.gitattributes

```diff
@@ -0,0 +1,13 @@
+.* export-ignore
+*.md export-ignore
+tests export-ignore
+docs export-ignore
+examples export-ignore
+mongo-orchestration export-ignore
+tools export-ignore
+Makefile export-ignore
```

Comment:
This file might just be a candidate for removal, per https://github.com/mongodb/mongo-php-library/pull/1021#discussion_r1057405814.